### ISSUE TITLE
198.105.244.228:80

### ISSUE THREAD

**Issue by gerroon** [y20k/transistor#167]
Hi

Is there any reason why this app constantly trying to send packets to this ip address? It even tries to connect to this when I am playing a stream from 127.0.0.1 (local music server).

This is not a disclosed behavior and  there is no way to disable whatever that is. Thankfully my firewall blocks all such requests by default.

You are also sending packets over http.

thanks

**Comment by y20k** [comment]
Hi there. 

Thank you for finding this! That is very strange behaviour. The playback in Transistor is entirely done by the open source library [ExoPlayer](https://github.com/google/ExoPlayer). I am not sending queries to the mentioned IP. 

I would like to recreate this behaviour. How do you watch network traffic on an Android device? I have honestly no clue. Could you post steps how to do that? And maybe a screenshot?

**Comment by gerroon** [comment]
Hi

I use https://github.com/ukanth/afwall/ and it has internal loggging which keeps a tap on the processes that are blocked by Iptables. 

You can try NetworkLog as well, meanwhile I will try it too.

Like I mentioned, Afwall blocks all traffic by Transistor (which might sound weird) because I do not need it to connect to any external stream, because it plays the stream from the local Mpd server which servers the stream over http://127.0.0.1:6600/stream.ogg

I will invgestigate the issue and get back to this bug, if necessary I will close it. I would not want to waste your time if it is just a logging problem. I just assumed that the app might have some analytics implemented.


thanks

**Comment by y20k** [comment]
Okay. Thanks for the info. Where did you get your copy of Transistor get from? And if not from the Google PlayStore, which version are you using? (Just to make sure that you did not get an evil twin of my app. *)

> I just assumed that the app might have some analytics implemented.

Nope. No analytics here.

(*) I meant that only half jokingly. My other app has an evil clone in the app store. The downside of open source I guess.

**Comment by gerroon** [comment]
I am sory to mentioned that I use the Fdroid version.

**Comment by y20k** [comment]
The most current version of Transistor is v3.0.2. F-Droid has v3.0.0. I [contacted](https://gitlab.com/fdroid/fdroiddata/issues/1091#note_58722824) them already: The last automatic build on F-Droid failed (the build their APKs directly from source). v3.0.2 should be available shortly. I doubt that that would make any difference. But it is good to know.

Lets hope that something is wrong with your logging. 

You could try to use the [latest version](https://github.com/y20k/transistor/releases/tag/v3.0.2) here on GitHub. That one got compiled on my system. If the GitHub version does not try to reach out and the F-Droid one does, it is time for conspiracy theories. 😎

**Comment by gerroon** [comment]
Thanks for the further steps. I will investigate it for sure.

**Comment by gerroon** [comment]
I installed the latest github. And this is really weird. 

I originally thought that maybe there is some radio information in the local stream which Transistor tries to grab extra radio information from. So I changed the radio in Mpd and cleared data readded and it still tries to connect to the ip I gave in the title. So it is not a stream information type of thing.

Then I thought I should try with another Mpd instance I have in my house (over local wireless), so I enabled Transistor to access to local wifi (it was blocked before) and I started another radio on it then added this http://192.168.x.x:6600/mpd.ogg. Transistor again tried to connect to the very same ip.

I did one last test. I added the same local stream to another player (which has only local and local lan access) that one does not try to connect any other ip other than the local stream. 

Basically I do not know why it tries to access that ip. But when I add the stream it says "download started" and that is when Afwall blocks it however I do not know what the latency is between blocking and showing a toast about it.

**Comment by y20k** [comment]
Just in case you had not already done this ... 

https://www.google.com/search?q=198.105.244.228

It got me similar problem descriptions. For example: [Internal.network is not resolving hosts and ping redirected to 198.105.244.22](https://www.linuxquestions.org/questions/linux-networking-3/internal-network-is-not-resolving-hosts-and-ping-redirected-to-198-105-244-228-a-4175528190/). I am not competent enough to judge (or to even understand) this. It looks like a problem that has to do with cable modems in the US??

**Comment by gerroon** [comment]
@y20k

Well I looked at the ip but I could not get much lead.

Lets say it is an ip resolving issue, the issue is that why would Transistor try to resolve a local ip address 127.0.0.1 and then not able resolve and redirect it to some unknown ip?

The p is owned by Searchguide it seems.

**Comment by y20k** [comment]
Sorry. I think I cannot help. I have no idea what causes that behavior. I will close this Issue. :-/
